spec/index.html

```diff
@@ -869,10 +956,21 @@ <h3>Reifying Triples</h3>
       a fresh RDF <a data-cite="RDF12-CONCEPTS#dfn-blank-node">blank node</a> is allocated
       and used to identify this relationship.</p>
 
+    <p class="note">When using RDF 1.2-specific features, such as
+      <a data-cite="RDF12-CONCEPTS#dfn-triple-term">triple terms</a>,
+      the specific RDF version should be announced using
+      either the <a href="#grammar-production-sparqlVersion"><code>VERSION</code></a>
+      or <a href="#grammar-production-version"><code>@version</code></a>
+      directives early in the document.
+      This allows parsers that do not support these features to detect
+      the presense of such features early.
+    </p>
+
     <pre id="ex-reified-triple"
          class="example turtle" data-transform="updateExample"
          title="Reified Triple">
       <!--
+      VERSION     "1.2"
```

Comment:
There was a suggestion in https://github.com/w3c/rdf-star-wg/issues/141 to allow `VERSION 1.2` (without quotes) as shorthand.
Do we want to introduce this in the spec?
If so, it may be useful to use this shorthand in all examples.

Reply:
I'll update the grammar terminal to allow `([0-9]+ '.' [0-9]+) | ('"' [^#x22#x5C#x0A#x0D] '"')` with the provision that the pure-decimal version be treated as a string when setting |curVersion|.